lpa-submissions-e2e-tests/cypress/integration/appealsDevelopmentPlan/appealsDevelopmentPlanSteps.js

```diff
@@ -95,13 +95,21 @@ Then(`the LPA Planning Officer is presented with the ability to provide informat
 
 Then(`progress is halted with an error message {string}`, (errorMessage) => {
   errorMessage === 'Select yes if there is a relevant Development Plan or Neighbourhood Plan'
-    ? cy.validateErrorMessage(errorMessage, '[data-cy="has-plan-submitted-error"]', 'has-plan-submitted')
-    : cy.validateErrorMessage(errorMessage, '[data-cy="plan-changes-text-error"]', 'plan-changes-text');
+    ? cy.validateErrorMessage(
+        errorMessage,
+        '[data-cy="has-plan-submitted-error"]',
+        'has-plan-submitted',
+      )
+    : cy.validateErrorMessage(
+        errorMessage,
+        '[data-cy="plan-changes-text-error"]',
+        'plan-changes-text',
+      );
 });
 
 Then(`the LPA Planning Officer remains on the page`, () => {
   cy.verifyPageHeading(pageHeading);
-  cy.verifyPageTitle(pageTitle);
+  cy.title().should('include', 'Error:');
```

Comment:
As discussed, I recommend we move this into the cy.validateErrorMessage command, that way we don't have to explicitly call it every time.

Reply:
Updated as per review comments